### PR TITLE
Add audio modal & button actions

### PR DESCRIPTION
**Pull request by Jigar-bhaliya**

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  time: PropTypes.number,
+  allowStartStopRecording: PropTypes.bool.isRequired,
+};
+
+const defaultProps = {
+  amIModerator: false,
+  record: false,
+  recording: false,
+  time: 0,
+};
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx]: Not needed

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+class RecordIndicator extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx:51]: empty line before

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+      time: (props.time ? props.time : 0),
+    };
+
+    this.incrementTime = this.incrementTime.bind(this);
+  }
+
+  componentDidUpdate() {
+    const { recording } = this.props;
+    if (!recording) {
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx:61]: empty line before

File: bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx [hunk 4]
```diff
@@ -1,18 +1,28 @@
...
-  <div className="w-1/6 mr-4">
-    <img src={avatar} className="fill-current" alt="" />
-  </div>
-);
+const UserAvatar = ({ color, children }) => {
+  const style = {
+    backgroundColor: color,
+    minWidth: '87px',
+    minHeight: '87px',
+  };
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx]: use tailwind

File: bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx [hunk 5]
```diff
@@ -1,18 +1,28 @@
...
 
 UserAvatar.defaultProps = {
-  avatar: '',
+  color: '',
 };
 
 UserAvatar.propTypes = {
-  avatar: PropsType.string,
+  color: PropsType.string,
 };
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx]: not needed

File: bigbluebutton-html5/imports/ui/components/app/AppView.jsx [hunk 6]
```diff
@@ -1,19 +1,25 @@
 import React from 'react';
-
+import ModalContainer from '../modal/container';
+import AudioContainer from '../audio/container';
 import Tabs from '../tabs';
-import Header from '../header';
-import Footer from '../footer';
+// import Header from '../header';
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/app/AppView.jsx]: remove dead code

File: bigbluebutton-html5/imports/ui/components/UserList/UserListView.jsx [hunk 7]
```diff
@@ -0,0 +1,25 @@
+import React, { Fragment } from 'react';
+
+import UserAvatar from '../UserAvatar';
+
+const UserListView = ({
+  users,
+}) => (
+  <Fragment>
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/UserList/UserListView.jsx]: Fragment is not required

File: bigbluebutton-html5/imports/ui/components/app/AppContainer.jsx [hunk 8]
```diff
@@ -1,11 +1,19 @@
 import React, { Component } from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import { withModalMounter } from '../modal/service';
 
 import AppView from './AppView';
 
-export default class AppContainer extends Component {
+class AppContainer extends Component {
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/app/AppContainer.jsx]: use functional component

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+  handleToggleMuteMicrophone: PropTypes.func.isRequired,
+  handleJoinAudio: PropTypes.func.isRequired,
+  handleLeaveAudio: PropTypes.func.isRequired,
+  disable: PropTypes.bool.isRequired,
+  muted: PropTypes.bool.isRequired,
+  showMute: PropTypes.bool.isRequired,
+  inAudio: PropTypes.bool.isRequired,
+  listenOnly: PropTypes.bool.isRequired,
+  talking: PropTypes.bool.isRequired,
+};
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx]: Not needed

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+class AudioControlsView extends PureComponent {
+  componentDidMount() {
+    const { processToggleMuteFromOutside } = this.props;
+    if (Meteor.settings.public.allowOutsideCommands.toggleSelfVoice
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx:11]: empty line before

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+      inAudio,
+      listenOnly,
+      isVoiceUser,
+      inputStream,
+      isViewer,
+      isPresenter,
+    } = this.props;
+
+    let joinIcon = 'telephone-off';
+    if (inAudio) {
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx:34]: empty line before

File: bigbluebutton-html5/imports/ui/components/audio/audio-modal/component.jsx [hunk 12]
```diff
@@ -262,7 +262,7 @@ class AudioModal extends Component {
     });
 
     return joinEchoTest().then(() => {
-      //console.log(inputDeviceId, outputDeviceId);
+      // console.log(inputDeviceId, outputDeviceId);
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/audio/audio-modal/component.jsx]: remove dead code

File: bigbluebutton-html5/imports/ui/components/common/Button/Button.jsx [hunk 13]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
+// import styles from './styles';
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/common/Button/Button.jsx]: remove

File: bigbluebutton-html5/imports/ui/components/common/Button/Button.jsx [hunk 14]
```diff
@@ -80,6 +84,7 @@ Button.propTypes = {
   color: PropTypes.string,
   variant: PropTypes.string,
   fontWeight: PropTypes.string,
+  onClick: PropTypes.func,
   children: PropTypes.node,
 };
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/common/Button/Button.jsx]: Remove defaultProps and propTypes

File: bigbluebutton-html5/imports/ui/components/common/IconButton/IconButton.jsx [hunk 15]
```diff
@@ -21,27 +22,41 @@ const IconButton = ({
...
   let buttonColor = 'bg-white hover:bg-gray-200';
-  let buttonSize = 'p-5 rounded-lg';
+  let buttonSize = 'min-h-64 min-w-64 rounded-lg';
   let buttonMargin = ' mr-3';
   let iconSize = ' w-6 h-6';
   let buttonTransparent;
+  let iconMinVh = {
+    minWidth: '20px',
+    minHeight: '20px',
+  };
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/common/IconButton/IconButton.jsx]: use tailwind class

File: bigbluebutton-html5/imports/ui/components/common/IconButton/IconButton.jsx [hunk 16]
```diff
@@ -21,27 +22,41 @@ const IconButton = ({
...
 
   if (size === SIZE_TYPE.SMALL) {
-    buttonSize = 'p-3 rounded-md';
+    buttonSize = 'min-h-40 min-w-40 rounded-md';
     buttonMargin = ' mr-2';
     iconSize = ' w-4 h-4';
+    iconMinVh = {
+      minWidth: '15px',
+      minHeight: '15px',
+    };
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/common/IconButton/IconButton.jsx]: use tailwind class

File: bigbluebutton-html5/imports/ui/components/footer/DesktopShare/DesktopShare.jsx [hunk 17]
```diff
@@ -0,0 +1,202 @@
...
+  intl: intlShape.isRequired,
+  amIPresenter: PropTypes.bool.isRequired,
+  handleShareScreen: PropTypes.func.isRequired,
+  handleUnshareScreen: PropTypes.func.isRequired,
+  isVideoBroadcasting: PropTypes.bool.isRequired,
+  screenSharingCheck: PropTypes.bool.isRequired,
+  screenShareEndAlert: PropTypes.func.isRequired,
+  isMeteorConnected: PropTypes.bool.isRequired,
+  screenshareDataSavingSetting: PropTypes.bool.isRequired,
+};
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/footer/DesktopShare/DesktopShare.jsx]: remove

File: bigbluebutton-html5/imports/ui/components/footer/DesktopShare/DesktopShare.jsx [hunk 18]
```diff
@@ -0,0 +1,202 @@
...
+    return false;
+  };
+
+  const shouldAllowScreensharing = screenSharingCheck
+        && !isMobileBrowser
+        && amIPresenter;
+
+  return shouldAllowScreensharing
+    ? (
+
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/footer/DesktopShare/DesktopShare.jsx]: remove empty line

File: bigbluebutton-html5/imports/ui/components/footer/FooterView.jsx [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+import DesktopShare from './DesktopShare';
+import AudioControlsContainer from '../audio/audio-controls/AudioControlsContainer';
+import JoinVideoOptionsContainer from '../video-provider/video-button/VideoButtonContainer';
+import VideoProviderContainer from '../video-provider/container';
+import { withModalMounter } from '/imports/ui/components/modal/service';
+import getFromUserSettings from '/imports/ui/services/users-settings';
+import RecordIndicatorContainer from '../RecordIndicator';
+import UserListContainer from '../UserList';
+
+
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/footer/FooterView.jsx:10]: remove empty line

File: bigbluebutton-html5/imports/ui/components/footer/FooterView.jsx [hunk 20]
```diff
@@ -0,0 +1,92 @@
...
+          />
+          <UserAvatar
+            avatar="images/user_2.png"
+          />
+          <UserAvatar
+            avatar="images/user_3.png"
+          />
+          <UserAvatar
+            avatar="images/user_4.png"
+          /> */}
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/footer/FooterView.jsx]: Remove dead code

File: bigbluebutton-html5/imports/ui/components/header/Header.jsx [hunk 21]
```diff
@@ -1,35 +1,65 @@
...
+  mountModal: PropTypes.func.isRequired,
+  amIModerator: PropTypes.bool,
+  isBreakoutRoom: PropTypes.bool,
+  isMeteorConnected: PropTypes.bool.isRequired,
+};
+
+Header.defaultProps = {
+  amIModerator: false,
+  isBreakoutRoom: false,
+};
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/header/Header.jsx]: remove

File: bigbluebutton-html5/imports/ui/components/header/HeaderContainer.jsx [hunk 22]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { withTracker } from 'meteor/react-meteor-data';
+import { meetingIsBreakout } from '/imports/ui/components/app/service';
+import { withModalMounter } from '../modal/service';
+import Service from '../actions-bar/service';
+
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/header/HeaderContainer.jsx]: remove empty line

File: bigbluebutton-html5/imports/ui/components/tabs/components/Tab/Tab.jsx [hunk 23]
```diff
@@ -11,7 +11,10 @@ const Tab = ({
   onClick,
 }) => {
   let tabClass = 'w-full';
-
+  const iconMinVh = {
+    minWidth: '20px',
+    minHeight: '20px',
+  };
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/tabs/components/Tab/Tab.jsx]: Tailwind class

File: bigbluebutton-html5/imports/ui/components/video-provider/video-button/VideoButtonView.jsx [hunk 24]
```diff
@@ -0,0 +1,85 @@
...
+    //   onClick={handleOnClick}
+    //   hideLabel
+    //   aria-label={intl.formatMessage(intlMessages.videoButtonDesc)}
+    //   color={hasVideoStream ? 'primary' : 'default'}
+    //   icon={hasVideoStream ? 'video' : 'video_off'}
+    //   ghost={!hasVideoStream}
+    //   size="lg"
+    //   circle
+    //   disabled={isDisabled}
+    // />
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/video-provider/video-button/VideoButtonView.jsx]: remove

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx [hunk 25]
```diff
@@ -15,7 +15,7 @@ import Button from '/imports/ui/components/button/component';
 const propTypes = {
   streams: PropTypes.arrayOf(PropTypes.object).isRequired,
   onMount: PropTypes.func.isRequired,
-  webcamDraggableDispatch: PropTypes.func.isRequired,
+  // webcamDraggableDispatch: PropTypes.func.isRequired,
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx]: remove

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx [hunk 26]
```diff
@@ -102,13 +102,13 @@ class VideoList extends Component {
...
-        value: this.grid,
-      },
-    );
+    // const { webcamDraggableDispatch } = this.props;
+    // webcamDraggableDispatch(
+    //   {
+    //     type: 'setVideoListRef',
+    //     value: this.grid,
+    //   },
+    // );
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx]: remove

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx [hunk 27]
```diff
@@ -149,12 +149,12 @@ class VideoList extends Component {
...
-        type: 'setOptimalGrid',
-        value: optimalGrid,
-      },
-    );
+    // webcamDraggableDispatch(
+    //   {
+    //     type: 'setOptimalGrid',
+    //     value: optimalGrid,
+    //   },
+    // );
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx]: remove

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx [hunk 28]
```diff
@@ -40,12 +40,12 @@ class VideoListItem extends Component {
...
-        type: 'setVideoRef',
-        value: this.videoTag,
-      },
-    );
+    // webcamDraggableDispatch(
+    //   {
+    //     type: 'setVideoRef',
+    //     value: this.videoTag,
+    //   },
+    // );
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx]: remove

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx [hunk 29]
```diff
@@ -171,10 +174,10 @@ class VideoListItem extends Component {
...
             className={cx({
               [styles.media]: true,
-              [styles.cursorGrab]: !webcamDraggableState.dragging
-                && !isFullscreen && !swapLayout,
-              [styles.cursorGrabbing]: webcamDraggableState.dragging
-                && !isFullscreen && !swapLayout,
+              // [styles.cursorGrab]: !webcamDraggableState.dragging
+              //   && !isFullscreen && !swapLayout,
+              // [styles.cursorGrabbing]: webcamDraggableState.dragging
+              //   && !isFullscreen && !swapLayout,
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx]: remove

File: bigbluebutton-html5/imports/ui/components/video-provider_default/video-list/video-list-item/component.jsx [hunk 30]
```diff
@@ -0,0 +1,235 @@
...
+VideoListItem.defaultProps = {
+  numOfStreams: 0,
+};
+
+VideoListItem.propTypes = {
+  actions: PropTypes.arrayOf(PropTypes.object).isRequired,
+  cameraId: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  numOfStreams: PropTypes.number,
+};
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/video-provider_default/video-list/video-list-item/component.jsx:235]: remove

Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/video-provider_default/video-list/video-list-item/component.jsx:235]: removed

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx (hunk 29 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx (hunk 28 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/video-provider/video-list/video-list-item/component.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx (hunk 27 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx (hunk 26 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx (hunk 25 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/video-provider/video-list/component.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/video-provider/video-button/VideoButtonView.jsx (hunk 24 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/video-provider/video-button/VideoButtonView.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/tabs/components/Tab/Tab.jsx (hunk 23 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/tabs/components/Tab/Tab.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/header/HeaderContainer.jsx (hunk 22 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/header/HeaderContainer.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/header/Header.jsx (hunk 21 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/header/Header.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/footer/FooterView.jsx (hunk 20 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/footer/FooterView.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/footer/FooterView.jsx (hunk 19 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/footer/FooterView.jsx:10]: removed

File: bigbluebutton-html5/imports/ui/components/footer/DesktopShare/DesktopShare.jsx (hunk 18 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/footer/DesktopShare/DesktopShare.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/footer/DesktopShare/DesktopShare.jsx (hunk 17 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/footer/DesktopShare/DesktopShare.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/common/IconButton/IconButton.jsx (hunk 16 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/common/IconButton/IconButton.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/common/IconButton/IconButton.jsx (hunk 15 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/common/IconButton/IconButton.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/common/Button/Button.jsx (hunk 14 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/common/Button/Button.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/common/Button/Button.jsx (hunk 13 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/common/Button/Button.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/audio/audio-modal/component.jsx (hunk 12 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/audio/audio-modal/component.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx (hunk 11 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx:34]: removed

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx (hunk 10 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx:11]: removed

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx (hunk 9 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/app/AppContainer.jsx (hunk 8 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/app/AppContainer.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/UserList/UserListView.jsx (hunk 7 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/UserList/UserListView.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/app/AppView.jsx (hunk 6 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/app/AppView.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx (hunk 5 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx (hunk 4 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx]: fixed

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx (hunk 3 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx:61]: removed

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx (hunk 2 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx:51]: removed

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx (hunk 1 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx]: removed

File: bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx [hunk 31]
```diff
@@ -1,18 +1,17 @@
...
-  avatar: '',
-};
-
-UserAvatar.propTypes = {
-  avatar: PropsType.string,
+const UserAvatar = ({ color, children }) => {
+  const style = {
+    backgroundColor: color,
+  };
+  return (
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx:8]: add empty line before

File: bigbluebutton-html5/imports/ui/components/app/AppContainer.jsx [hunk 32]
```diff
@@ -1,11 +1,9 @@
...
 import AppView from './AppView';
 
-export default class AppContainer extends Component {
-  render() {
-    return (
-      <AppView />
-    );
-  }
-}
+const AppContainer = ({ props }) => <AppView {...props} />;
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/app/AppContainer.jsx]: should be like

`const AppContainer = (props) => <AppView {...props} />;`

File: bigbluebutton-html5/imports/ui/components/tabs/components/Tab/Tab.jsx [hunk 33]
```diff
@@ -11,7 +9,7 @@ const Tab = ({
   onClick,
 }) => {
   let tabClass = 'w-full';
-
+  const iconMinVh = 'min-w-20 min-h-20';
   if (index === activeKey) {
```
Comment (piyushdhamecha) [inline bigbluebutton-html5/imports/ui/components/tabs/components/Tab/Tab.jsx:14]: add empty line before

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx (hunk 2 above)
Comment (piyushdhamecha) [reply bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx:50]: Add an empty line before not remove

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx (hunk 3 above)
Comment (piyushdhamecha) [reply bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx:61]: Add an empty line before not remove

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx (hunk 2 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx:51]: done

File: bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx (hunk 3 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/RecordIndicator/RecordIndicatorView.jsx:61]: done

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx (hunk 10 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx:11]: done

File: bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx (hunk 11 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/audio/audio-controls/AudioControlsView.jsx:34]: done

File: bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx (hunk 31 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/UserAvatar/UserAvatar.jsx:8]: done

File: bigbluebutton-html5/imports/ui/components/app/AppContainer.jsx (hunk 32 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/app/AppContainer.jsx]: done

File: bigbluebutton-html5/imports/ui/components/tabs/components/Tab/Tab.jsx (hunk 33 above)
Comment (Jigar-bhaliya) [reply bigbluebutton-html5/imports/ui/components/tabs/components/Tab/Tab.jsx:14]: done

**Pull request merged**